src/members/Member.css

```diff
@@ -47,4 +47,10 @@ label {
     margin: 3em 3em;
     padding: 2em;
   }
-  
\ No newline at end of file
+  
+  @media only screen and (max-width: 644px) {
+  .container-members {
+    overflow: hidden;
+    overflow-x: scroll;
+  }
+}
```

Comment:
Add a new line at the end.

Reply:
> Add a new line at the end.

Hi @codesankalp  , i am not able to add new line in the end , is this due to 'Prettier' ?